wire/internal/wire/testdata/Chain/pkg

```diff
@@ -1 +1 @@
-foo
+example.com/foo
```

Comment:
Can you explain why all of these need to change to example.com?

Reply:
Added comments in code.

This is the place the test code copied the packages to. I could add the module path when copying the code, is that preferred?

Reply:
Right, but why? It's not required for go 1.11 that everything be under a domain, right?

I.e., why not leave it so that the code is just in package "foo" and don't change the test code to copy the packages to example.com/foo?

Reply:
Some of the test cases contain more than one packages (foo and bar ...), it is better we put them in one module.

Reply:
Why? It just seems to me like it's making the whole test suite look like it requires modules, which it shouldn't; it should continue to work with go 1.10.